LandsitesTools/interface_settings.py

```diff
@@ -0,0 +1,83 @@
+import re
+from configparser import ConfigParser, ExtendedInterpolation
+from pathlib import Path
+import pandas as pd
+from dateutil.parser import parse as parsetime
+
+
+def splitSequence(sequence, sep_regex='[ ,;]+'):
```

Comment:
Small question: the PEP-8 guidelines recommend using lower-case letters and underscores for function names: https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Of course, camel-case is also completely fine, IMO we should just agree on one common scheme :-)

Reply:
I think it is a good idea to follow PEP-8 conventions as @lasseke suggests. Let us decide tomorrow together, as anyway other changes may be needed to make the settings interface work properly with interactive modifications.